### PR TITLE
docs: fix references to upstreams in union example

### PR DESCRIPTION
**Pull request by sntran**
#### What is the purpose of this change?

Updates documentation for Union remote to reflect the correct paths and upstreams

#### Was the change discussed in an issue or in the forum before?

No.

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/union.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Storage> union
 List of space separated upstreams.
 Can be 'upstreama:test/dir upstreamb:', '\"upstreama:test/space:ro dir\" upstreamb:', etc.
 Enter a string value. Press Enter for the default ("").
-upstreams>
+upstreams> C:\dir1 C:\dir2 C:\dir3
```
Comment (ncw) [inline docs/content/union.md:122]: I don't like this example on the local disk - can we use `mydrive:private/backup` like the example above?

**Review by ncw — COMMENTED** [review]
Great spotting on the text! See inline for a comment on the `upstreams>` help

File: docs/content/union.md (hunk 1 above)
Comment (sntran) [reply docs/content/union.md:122]: I based this on the following examples when the usage of the remote are showed.

image

If you want to change this, we will need to change those examples too.

Comment (ncw) [reply docs/content/union.md:122]: Ah! Yes we probably should as `C:\dir1` is particularly confusing as it is a Windows path, but looks kind of like an rclone remote.

Maybe make 3 remotes for examples and stick to those?  s3:dir1 gdrive:dir2 dropbox:dir3 or something like that? (to pick 3  popular backends!)

**Comment by ivandeex** [comment]
@sntran Are you still with us?

> Maybe make 3 remotes for examples and stick to those? s3:dir1 gdrive:dir2 dropbox:dir3 or something like that? (to pick 3 popular backends!)

**Comment by sntran** [comment]
Hi, yes.

I was thinking about the @ncw's comment, but I was not sure whether to proceed as he suggested, and I lost track of time to other projects.

I think the reason the examples used local remotes was to simplify the config, and I think it's the correct approach. We want to show example of union remote, not how to create different types of remote. Switching to use other types of remote would make the example config long.

File: docs/content/union.md (hunk 1 above)
Comment (ncw) [reply docs/content/union.md]: I've fixed this up now

**Comment by ncw** [comment]
I've fixed this up so we can merge this change - thank you very much @sntran 

Do send a followup if you think it can be improved further.

**Pull request merged**